docs/doc_1_concepts.md

```diff
@@ -59,3 +59,44 @@ If you belive a method should be marked as stable, feel free to raise a PR appen
 ## Example
 
 To learn more about using the SDK, see the [basic example](https://github.com/mongodb/atlas-sdk-go/blob/main/examples/basic/basic.go).
+
+## Release Strategy (Semantic Versioning)
+
+The Atlas Go SDK follows the Semantic Versioning (SemVer) specification. The versioning scheme used for this SDK is as follows: `vYYYYMMDDXXX.Y.Z`, where:
+
+- `YYYYMMDDXXX` represents the major version number and includes the date of the [Versioned API Resource Version](https://www.mongodb.com/docs/atlas/api/versioned-api-overview/) used in the SDK followed by three digits for other possible breaking changes.
+- `Y` represents the minor version number, indicating non-breaking iterations of the same Versioned API Resource.
+- `Z` represents the patch version number, indicating non-user-affecting fixes in the SDK.
+
+### Versioning Rules
+
+#### Major Version (vYYYYMMDDXXX.0.0)
+
+A major version increment signifies breaking changes in the SDK. The rules for major version increments are as follows:
+
+1. When MongoDB introduces a new version of the Versioned API or the SDK undergoes breaking changes, increment the major version. For example, `vYYYYMMDD` will become `v20300101`.
```

Comment:
When MongoDB introduces a new *resource* version of the Versioned API or the SDK undergoes breaking changes, increment the major version.
* For new API resource versions, formatted as `vYYYYMMDD`, the SDK major version becomes `vYYYYMMDD001`. For example, `v20300101` becomes `v20300101001`.
* To introduce SDK-level breaking changes on an already existing API resource version, the `XXX` part of the SDK's major version is incremented by 1. For example, `v20300101001` becomes `v20300101002`.